table-topic-solutions/playground/append-scenario/schemas/OrderV3.avsc

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "record",
+  "name": "Order",
+  "namespace": "demo.append",
+  "fields": [
+    {"name": "order_id", "type": "long"},
+    {"name": "product_name", "type": "string"},
+    {"name": "order_description", "type": ["null", "string"], "default": null},
+    {"name": "f_v3", "type": ["null", "string"], "default": null}
```

Comment:
in my example I used obligatory field - I think I tried to specify as optional first and later redefine to obligatory but that indeed would be incompatible schema evolution. (not sure if that will matter but just mentioning)